### PR TITLE
Fixing some issues with the Verify option in test client.

### PR DESCRIPTION
**Pull request by simoncowie**
Been using the test client recently and wanted to add some fixes we found were necessary.

### CODE REVIEW

File: src/EventStore.TestClient/Commands/DvuBasic/DvuBasicProcessor.cs [hunk 1]
```diff
@@ -208,7 +208,7 @@ private void Write(Status status, int writerIdx, CommandProcessorContext context
 
             var failed = 0;
 
-            var rnd = new Random();
+            var rnd = new Random(writerIdx);
```
Comment (simoncowie) [inline src/EventStore.TestClient/Commands/DvuBasic/DvuBasicProcessor.cs]: The writers were getting a lot of version conflicts as random without seed would spit out the same values

File: src/EventStore.TestClient/Commands/DvuBasic/DvuBasicProcessor.cs [hunk 2]
```diff
@@ -296,6 +293,7 @@ private void Write(Status status, int writerIdx, CommandProcessorContext context
                                         wrongExpectedVersion, streamsDeleted, failed, requests);
             status.FinilizeStatus(writerIdx, failed != sent);
             connection.Close();
+            finish.Set();
```
Comment (simoncowie) [inline src/EventStore.TestClient/Commands/DvuBasic/DvuBasicProcessor.cs]: With `finish.Set()` called in the packageHandler callback, the last writer to finish would sometimes miss calling the `status.FinilizeStatus` method before results were tallied

**Comment by gregoryyoung** [comment]
This looks reasonable.

**Comment by pgermishuys** [comment]
Thank you for the PR @simoncowie. The wercker build is failing fetching sources, asked the guys from `wercker` to look into that.

**Pull request merged**